.github/workflows/command-l10n-update.yml

```diff
@@ -0,0 +1,117 @@
+name: Command L10n Update
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  init:
+    runs-on: ubuntu-latest
+
+    # On pull requests and if the comment starts with `/l10n update`
+    if: github.event.issue.pull_request != '' && startsWith(github.event.comment.body, '/l10n update')
+
+    outputs:
+      git_path: ${{ steps.git-path.outputs.path }}
+      arg1: ${{ steps.command.outputs.arg1 }}
+      arg2: ${{ steps.command.outputs.arg2 }}
+      head_ref: ${{ steps.comment-branch.outputs.head_ref }}
+
+    steps:
+      - name: Check actor permission
+        uses: skjnldsv/check-actor-permission@v2
+        with:
+          require: write
+
+      - name: Add reaction on start
+        uses: peter-evans/create-or-update-comment@v2
+        with:
+          token: ${{ secrets.COMMAND_BOT_PAT }}
+          repository: ${{ github.event.repository.full_name }}
+          comment-id: ${{ github.event.comment.id }}
+          reaction-type: "+1"
+
+      - name: Parse command
+        uses: skjnldsv/parse-command-comment@master
+        id: command
+
+      # Init path depending on which command is run
+      - name: Init path
+        id: git-path
+        run: |
+          if ${{ startsWith(steps.command.outputs.arg1, '/') }}; then
+            echo "::set-output name=path::${{ github.workspace }}${{steps.command.outputs.arg1}}"
```

Comment:
`set-output` is deprecated btw :)
https://github.com/nextcloud/.github/blob/88a5069d2475125efcd3b352192e1ce0dfc14e0f/workflow-templates/appstore-build-publish.yml#L31-L32

Reply:
done